drape/stipple_pen_resource.hpp

```diff
@@ -24,7 +22,9 @@ using PenPatternT = buffer_vector<uint16_t, 2>;
 
 inline uint16_t PatternFloat2Pixel(double d)
 {
-  return static_cast<uint16_t>(std::round(d));
+  auto const px = static_cast<uint16_t>(std::round(d));
```

Comment:
Isn't the result of `std::round` equal to the result of `static_cast<uint16_t>`?

Reply:
No